config/env.development

```diff
@@ -231,7 +231,7 @@ MOCK_ELASTIC=
 MOCK_REDIS=
 
 # Elasticsearch info
-ELASTIC_URL=http://127.0.0.1
+ELASTIC_URL=http://localhost
```

Comment:
#2990, which had this same fix, was temporarily closed.
Are you still having the same issue?